styles/components/panel.scss

```diff
@@ -0,0 +1,163 @@
+$border-color: #ddd;
+
+.panel {
+	background: $brand-white;
+	border-radius: 5px;
+	box-shadow: 0 1px 1px rgba(0,0,0,.05);
+	ul { margin-top: 0; }
+	.list {
+		padding-left: 0;
+		margin-bottom: 0;
+		list-style-type: none;
+		.list-item {
+			border-width: 1px 0;
+			border-radius: 0;
+			position: relative;
+			display: block;
+			padding: 10px 15px;
+			margin-bottom: -1px;
+			background: $brand-white;
+			border: 1px solid $border-color;
+		}		
+	}
+}
+
+.panel-default {
```

Comment:
- We're not following bootstrap's conventions. We're following [**rscss**](https://github.com/rstacruz/rscss).